demos/covid-19/covid_19_seir/config.py

```diff
@@ -0,0 +1,22 @@
+MONTE_CARLO_RUNS = 1 # N monte carlo runs
```

Comment:
You're declaring the number of monte carlo runs, but not using this in any place. Either you can declare this in the sim_params file, what makes a lot of sense, or just use a hard coded number like you did.

Reply:
You're right, it was supposed to use the MONTE_CARLO_RUNS constant as a simulation parameter.